src/components/layout/navbar-items.tsx

```diff
@@ -61,17 +62,13 @@ export const NavbarItems: FC = () => {
                     }}
                 />
             </Group>
-            <Button
-                fullWidth
-                className={styles.hover}
-                justify="flex-start"
-                variant="transparent"
+            <NavLink
+                label="Logout"
+                leftSection={<SignOut />}
```

Comment:
we will want to add the active prop here too just for styling (it seems everything is white on hover without it)
```active={pathname === currentDashboardUrl}```